test/support/pages/discover_page.js

```diff
@@ -213,24 +213,6 @@ define(function (require) {
       .getProperty('value');
     },
 
-    clickLegendExpand: function clickLegendExpand() {
```

Comment:
I think here we should just adjust selectors on these tests and not remove them completely.